ziti/cmd/ascode/importer/importer_auth_policies.go

```diff
@@ -117,7 +117,7 @@ func (importer *Importer) ProcessAuthPolicies(input map[string][]interface{}) (m
 			}
 		}
 
-		if importer.loginOpts.Verbose {
+		if importer.LoginOpts.Verbose {
```

Comment:
i think this should just be a call to Debug not Info and doesn't need to bre wrapped in the if statement.